engine/runtime/src/main/scala/org/enso/interpreter/instrument/job/EnsureCompiledJob.scala

```diff
@@ -354,6 +371,27 @@ final class EnsureCompiledJob(protected val files: Iterable[File])
     else
       CompilationStatus.Success
 
+  private def setCacheWeights()(implicit ctx: RuntimeContext): Unit = {
+    ctx.contextManager.getAll.values.foreach { stack =>
```

Comment:
could we rename `getAll` to something that says what does it return? For example `getAllVisualisations` on `contextManager`helps to understand what we are getting.